### PR TITLE
Fixed replaced VMPs not getting new mappings

### PR DESCRIPTION
**Pull request by ekorchmar**

### CODE REVIEW

File: dmd/postprocessing.sql [hunk 1]
```diff
@@ -250,17 +254,8 @@ BEGIN
...
-	DO $_$
-	BEGIN
-		PERFORM VOCABULARY_PACK.DeleteAmbiguousMAPSTO();
-	END $_$;
 ;
 update concept_stage set concept_name = trim(concept_name)
-;
\ No newline at end of file
+;
+select devv5.genericupdate()
```
Comment (dimshitc) [inline dmd/postprocessing.sql:260]: Why is it here???

**Review by dimshitc — CHANGES_REQUESTED** [review]
As usual:
Put manual tables to Google drive, put the link to the readme
+ remove generic_update

File: dmd/postprocessing.sql (hunk 1 above)
Comment (ekorchmar) [reply dmd/postprocessing.sql:260]: it's how it was on my PC. fixing now

**Comment by Alexdavv** [comment]
@TinyRickC137 Do we consider this in the current dmd development?

**Comment by TinyRickC137** [comment]
@Alexdavv yes, I took all the available branches

**Pull request closed without merging**